openedxscorm/scormxblock.py

```diff
@@ -1,3 +1,5 @@
+from io import BytesIO
+from urllib.parse import urlparse
```

Comment:
`from` imports should be at the end?

Reply:
![image](https://user-images.githubusercontent.com/42294172/95086780-52962000-073a-11eb-8696-b426935dbad2.png)
just following the convention that is being followed in this xblock already